feature/default-branch-switching.feature

```diff
@@ -80,6 +82,7 @@ Feature: Default branch switching
       | 1.2.x  |
       | 1.3.x  |
       | 2.0.x  |
+    And branch "1.3.x" should branch from "1.2.x"
```

Comment:
This condition is currently violated.

Reply:
Note: this isn't run, right now, BTW - `@manually-tested` above

We'd need to unit test this, probably, or add automation (which I was too lazy to do -.- my bad), since these `.feature` cases only act as documentation.